soc/riscv/riscv-privileged/qingke/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The QingKe is SoC __core__ name.
Isn't the SoC name ch5xx?

Reply:
I believe "QingKe" is used both as the name for the their RISC-V core (usually referred to as "QingKe Vxx cores") and the RISC-V product line (usually referred to as "QingKe MCU series").

In summary, WCH has three product lines: "CH32Fxxx" for Cortex-M cores with STM32-compatible peripherals, designed as a pin-to-pin replacement for STM32; "CH32Vxxx" for "QingKe RISC-V core" with STM32-compatible peripherals; and "CH5xx" for "QingKe RISC-V core" with self-developed peripherals.

Reply:
If so as, it should be better to use "wch_ch5xx" here.
The directories is classified by product name.
And append manufacturer name as prefix.

Reply:
Thank you for your suggestion. So, if someone wants to contribute support for the CH32V series in the future, would it be appropriate to create a separate `wch_ch32v` directory? In the directory, there's a `Kconfig.series.qingke_v3`. Originally, my idea was that we could add `Kconfig.series.qingke_v4`, `Kconfig.series.qingke_v2`, and so on in the future. These different `series` use various combinations of `RISCV_ISA_EXT_*` flags, and these configurations would also be needed for the CH32V.

Reply:
After doing some research (and asking a friend working at WCH), I think I should pick `wch_ch32` as the soc name. The CH5xx (as well as the later CH6xx) are in fact a special enhanced model of their CH32V common MCU series.

Reply:
Sorry for late response.

> After doing some research (and asking a friend working at WCH), I think I should pick wch_ch32 as the soc name. The CH5xx (as well as the later CH6xx) are in fact a special enhanced model of their CH32V common MCU series.

It sounds good.
However, is the `wch_ch32` is not `wch_ch32v`?
WCH has `ch32` series as ARM product line. https://www.wch-ic.com/products/CH32F103.html
I think it needs to append `v` postfix.
WCH has `ch32` series as ARM product line. https://www.wch-ic.com/products/CH32F103.html

Reply:
You're right. Updated.